client/src/legacy/GridField.js

```diff
@@ -172,11 +172,14 @@ $.entwine('ss', function($) {
       );
     },
 
+    // Adds an actions column for the search action if it does not exist already
     fixColumns: function (visible) {
       this.find('.sortable-header').append('<th class="main col-Actions" />');
       this.find('tbody tr').each(function () {
         var cell = $(this).find('td:last');
-        cell.attr('colspan', 2);
+        // Note we need to add 1 to the current column span, because we're going to add a new column
+        var colspan = cell.attr('colspan') ?? 1;
+        cell.attr('colspan', +colspan + 1);
```

Comment:
That syntax seems a bit weird - if this is just to cast to number, please use `Number` instead.
```suggestion
        cell.attr('colspan', Number(colspan) + 1);
```